php-fpm/README.md

```diff
@@ -56,8 +56,42 @@ image](https://hub.docker.com/_/php).
 
 <!-- Optional -->
 ### Deploy with Kubernetes
-
-
+This image can also be deployed on a Kubernetes cluster, such as
+[minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).The
+following example YAML files are provided in the repository as
+reference for Kubernetes deployment:
+
+   * [`pv-local.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/php-fpm/pv-local.yaml), [`pvc-local.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/php-fpm/pvc-local.yaml):
+     local persistent volumes for databse and wordpress.
```

Comment:
is this related with Wordpress?

Reply:
updated. please review the new submit https://github.com/clearlinux/dockerfiles/pull/256